api/v1/types.proto

```diff
@@ -63,3 +63,9 @@ message LogicalVolumeGroup {
     repeated string VolumeRefs = 5;
     string Status = 6;
 }
+
+message CSINode {
```

Comment:
I think we need to add one more field here in which CSINode status will be stored:
`Status` - online/offline. 
Controller will use that field for cases when node will be replaced or accidentally goes down:
- some node goes down, e.g. NODE1
- controller set corresponding CR status to offline: CSINode.Spec.Status = "OFFLINE"
- new node is added to the cluster, new node has same hostname and ips that "old" node had
- controller detect that new node is corresponding to "old" node and change CSINOde.Spec.Status to "ONLINE" and restore node ID for new node based on value of CSINode.Spec.ID

Reply:
Don't think that we need to track node status here.

Reply:
What operator should to do when node goes down?